python/doc/examples/functional_modeling/field_functions/plot_process_sample.py

```diff
@@ -0,0 +1,160 @@
+"""
+Create a process sample from a sample
+=====================================
+"""
+# sphinx_gallery_thumbnail_number = 3
+# %%
+
+# %%
+# In this example, a :class:`~openturns.ProcessSample` is created from data.
+# The purpose is to illustrate how to create a :class:`~openturns.ProcessSample` from an already available sample of a field.
+# In addition, the computation of statistics of the process sample is presented.
+# The :class:`~openturns.ProcessSample` is defined over a :class:`~openturns.Mesh`.
+#
+# For this example, the realizations of a stochastic process are obtained using the Chaboche model.
+# A detailed explanation of this mechanical law is presented :ref:`here <use-case-chaboche>`.
+# In this example, the model is defined such that:
+# :math:`\sigma=g(\vect{X},\epsilon)`
+# with :math:`\vect{X}=[R,C,\gamma]^T` a vector of random variables and
+#
+# - the strain :math:`\epsilon\in[0.,0.07]`
+# - the stress :math:`\sigma`
+#
+
+# %%
+import openturns as ot
+import openturns.viewer as viewer
+from openturns.usecases import chaboche_model
+
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# Define the model and the mesh
+# -----------------------------
+
+# %%
+# First, the Chaboche model is loaded.
+
+# %%
+cm = chaboche_model.ChabocheModel()
+
+# %%
+# Then, a mesh using a :class:`~openturns.RegularGrid` is defined for the strain :math:`\epsilon\in[0.,0.07]` with :math:`N=100` vertices.
+
+# %%
+N = 100
+mesh = ot.RegularGrid(cm.strainMin, (cm.strainMax - cm.strainMin) / N, N)
+vertices = mesh.getVertices()
+
+# %%
+# Generate one sample of the field
+# --------------------------------
+
+# %%
+# One sample of the field is obtained by evaluating the Chaboche model on the mesh vertices
+# for one specific realization of the input vector :math:`\vect{X}_0=[700\times 10^6,2500\times 10^6,8.]^T`.
+
+# %%
+R = 700e6
+C = 2500e6
+gamma = 8.0
+X0 = [R, C, gamma]
+X_indices = [1, 2, 3]
+f = ot.ParametricFunction(cm.model, X_indices, X0)
+Y = [f(vertices[i])[0] for i in range(N)]
+
+# %%
+# Let's visualize this sample of the field.
+
+# %%
+graph = ot.Graph("One realization of the stochastic process", "Strain", "Stress (Pa)", True, "")
+curve = ot.Curve(vertices.asPoint(), Y)
+graph.add(curve)
+view = viewer.View(graph)
+
+# %%
+# The distribution of the input vector :math:`\vect{X}` is defined.
+
+# %%
+X_distribution = ot.JointDistribution([cm.R, cm.C, cm.Gamma])
+
+# %%
+# Generate different samples of the field
+# ---------------------------------------
+
+# %%
+# 50 samples of the input vector :math:`\vect{X}` are generated.
+
+# %%
+n_samples = 50
+X_samples = X_distribution.getSample(n_samples)
+
+# %%
+# The Chaboche model is evaluated for each of the input vector samples.
+
+# %%
+Y_list = []
+for i in range(n_samples):
+    f = ot.ParametricFunction(cm.model, X_indices, X_samples[i, :])
+    Y = [[f(vertices[i])[0]] for i in range(N)]
```

Comment:
idem

Reply:
Thank you, the modification has been made.